00XX-common-ledger-protocol/00XX-common-ledger-protocol.md

```diff
@@ -0,0 +1,273 @@
+# Common Ledger Protocol (CLP)
+
+## Preface
+
+This document describes the Common Ledger Protocol (CLP), a ledger protocol for
+bilateral liquidity relationships. It is similar in function to [Plugin RPC],
+but has been rewritten to use OER instead of JSON, and extensibility features
+have been added.
+
+## Introduction
+
+### Motivation
+
+There are many different types of ledgers, and Interledger aims to interoperate
+all of them. For fast ledgers that support hashlocks, it is enough to wrap the
+ledger's API in a [Ledger Plugin]. These allow you to send interledger payments
+through anyone else on that ledger who is running your plugin.
+
+In lots of scenarios, we don't have an underlying ledger that's fast enough to
+do every ILP payment on-ledger. There are also many situations when two parties
+trust one another up to an amount, and don't need any third-party ledger to
```

Comment:
These can't be disjoint, CLP is useless unless you have **both** a ledger that's slower than direct CLP **and** at least a little bit of off-ledger trust.